debian/copyright

```diff
@@ -0,0 +1,34 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: nemo-qml-plugin-dbus
+Source: https://git.sailfishos.org/mer-core/nemo-qml-plugin-dbus
+
+Files: *
+Copyright: 2013-2021 Jolla OY & many contributors (see git log)
+License: LGPL-2.1
+
+Files: debian/*
+Copyright: 2021-2021 Sebastian Spaeth <Sebastian@SSpaeth.de>
+License: GPL-3+
```

Comment:
Not sure would i like to add gpl3 to the repository just for packaging side. For that purpose it shouldn't be much better than v2?

Reply:
The debian directory does not need to declare the license separately, I deleted this statement